tests/ImageSharp.Tests/Formats/Jpg/JpegDecoderTests.cs

```diff
@@ -1,61 +1,62 @@
-    using SixLabors.ImageSharp.Formats.Jpeg.PdfJsPort;
-    using SixLabors.ImageSharp.Memory;
-    using SixLabors.ImageSharp.PixelFormats;
-    using SixLabors.ImageSharp.Tests.Formats.Jpg.Utils;
-    using SixLabors.ImageSharp.Tests.TestUtilities.ImageComparison;
-    using SixLabors.Primitives;
-
-    using Xunit;
-    using Xunit.Abstractions;
-
     // TODO: Scatter test cases into multiple test classes
     public class JpegDecoderTests
     {
         public static string[] BaselineTestJpegs =
-            {
-                TestImages.Jpeg.Baseline.Calliphora,
-                TestImages.Jpeg.Baseline.Cmyk,
-                TestImages.Jpeg.Baseline.Ycck,
-                TestImages.Jpeg.Baseline.Jpeg400,
-                TestImages.Jpeg.Baseline.Testorig420,
+        {
+            TestImages.Jpeg.Baseline.Calliphora,
+            TestImages.Jpeg.Baseline.Cmyk,
+            TestImages.Jpeg.Baseline.Ycck,
+            TestImages.Jpeg.Baseline.Jpeg400,
+            TestImages.Jpeg.Baseline.Testorig420,
                 
-                // BUG: The following image has a high difference compared to the expected output:
-                //TestImages.Jpeg.Baseline.Jpeg420Small,
-
-                TestImages.Jpeg.Baseline.Jpeg444,
-                TestImages.Jpeg.Baseline.Bad.BadEOF,
-                TestImages.Jpeg.Issues.MultiHuffmanBaseline394,
-                TestImages.Jpeg.Baseline.MultiScanBaselineCMYK,
-                TestImages.Jpeg.Baseline.Bad.BadRST
-            };
+            // BUG: The following image has a high difference compared to the expected output:
+            // TestImages.Jpeg.Baseline.Jpeg420Small,
+
+            TestImages.Jpeg.Baseline.Jpeg444,
+            TestImages.Jpeg.Baseline.Bad.BadEOF,
+            TestImages.Jpeg.Issues.MultiHuffmanBaseline394,
+            TestImages.Jpeg.Baseline.MultiScanBaselineCMYK,
+            TestImages.Jpeg.Baseline.Bad.BadRST
+        };
 
         public static string[] ProgressiveTestJpegs =
-            {
-                TestImages.Jpeg.Progressive.Fb, TestImages.Jpeg.Progressive.Progress,
-                TestImages.Jpeg.Progressive.Festzug, TestImages.Jpeg.Progressive.Bad.BadEOF,
-                TestImages.Jpeg.Issues.BadCoeffsProgressive178,
-                TestImages.Jpeg.Issues.MissingFF00ProgressiveGirl159,
-                TestImages.Jpeg.Issues.BadZigZagProgressive385,
-                TestImages.Jpeg.Progressive.Bad.ExifUndefType
-            };
+        {
+            TestImages.Jpeg.Progressive.Fb, TestImages.Jpeg.Progressive.Progress,
+            TestImages.Jpeg.Progressive.Festzug, TestImages.Jpeg.Progressive.Bad.BadEOF,
+            TestImages.Jpeg.Issues.BadCoeffsProgressive178,
+            TestImages.Jpeg.Issues.MissingFF00ProgressiveGirl159,
+            TestImages.Jpeg.Issues.BadZigZagProgressive385,
+            TestImages.Jpeg.Progressive.Bad.ExifUndefType
+        };
+
+        public static string[] FalsePositiveIssueJpegs =
```

Comment:
I fail to follow why did you named this "false positive".

Reply:
Because the images are actually fine. I checked them with jpeg snoop and there were no errors.

Reply:
Not all of the images labeled as "issues" are actually invalid. I meant this to indicate a regression test based on an GH issue. We may need to find a better naming for the whole thing.

Reply:
I didn't realize that. Thought they were all buggered.

I want to do a big refactor of the unit tests, grouping everything properly into namespaces etc.